data/transactions/logic/eval.go

```diff
@@ -533,6 +533,28 @@ const (
 // StackTypes is an alias for a list of StackType with syntactic sugar
 type StackTypes []StackType
 
+func parseStackTypes(spec string) StackTypes {
+	if spec == "" {
+		return nil
+	}
+	types := make(StackTypes, len(spec))
+	for i, letter := range spec {
+		switch letter {
+		case 'a':
+			types[i] = StackAny
+		case 'b':
+			types[i] = StackBytes
+		case 'i':
+			types[i] = StackUint64
+		case 'x':
+			types[i] = StackNone
+		default:
+			panic(spec)
```

Comment:
@jannotti Optional:  I'm imagining added context may simplify debugging.  Do you feel it's a worthwhile change to highlight the failing letter and possibly index?

If you prefer as is, feel welcomed to resolve the thread.

Reply:
For things that will panic at program start time, I don't worry too much about adding details, but I can certainly see the draw. The spec will have the opcode and name, which will lead to the line in opcodes.go pretty quickly.

Reply:
@jannotti Got it - I interpreted similarly though thought it prudent to double check.  Works for me as is - resolving.